minimap.js

```diff
@@ -39,10 +39,18 @@ pc.script.create('minimap', function (context) {
             "   visibility: visible;",
             "}"
         ].join('\n');
-        
+
         var style = document.createElement('style');
         style.innerHTML = css;
         document.querySelector('head').appendChild(style);
+
+        var uri = new pc.URI(window.location.href);
+        var query = uri.getQuery();
+
+        if ('multi' in query) {
+            this.canvas.style.display = 'none';
```

Comment:
can you not just not [append the canvas to `document.body`](#diff-339e2e232e6ed934e6b7d020a6bb4be1R12)?